### DISCUSSION TITLE
Show diagnostics in a floating window

### DISCUSSION THREAD

**Discussion by latrovalievioan** [LunarVim/LunarVim#3830] [Q&A] [answered]
Hi guys! I want to have my diagnostics in a floating window when hovering over the underlined code, so far I've added `lvim.lsp.diagnostics.virtual_text = false` to my config but this just disables the inline diagnostics. Can somebody give me a hand?

**Comment by LostInTheLogs** [comment, marked as answer]
make an autocmd on CursorHold and call `vim.diagnostic.open_float()` there

**Reply by latrovalievioan** [reply]
Thanks, I've figured it out. For anyone wondering, add this to `config.lua`:
`vim.cmd [[autocmd CursorHold,CursorHoldI * lua vim.diagnostic.open_float(nil, {focus=false})]]`
